service/worker/parentclosepolicy/workflow.go

```diff
@@ -24,6 +24,8 @@ import (
 	"context"
 	"time"
 
+	"github.com/uber/cadence/common/types/mapper/thrift"
```

Comment:
nit: this seems like it is not used anywhere

Reply:
Good catch. Removed.